backend/lib/data/projects_data.json

```diff
@@ -0,0 +1,61 @@
+[
+  {
+    "title": "Loot",
+    "nft_collection": {
+      "collection_url": "https://opensea.io/collection/lootproject"
+    },
+    "smart_contract": {
+      "contract_url": "https://etherscan.io/address/0xff9c1b15b16263c61d017ee9f65c50e4ae0113d7",
```

Comment:
this feels duplicative. is simply `contract_address` fine? with that we can derive the etherscan address

Reply:
yes, but what if we'll need to add more fields? This is broken down into associations. Maybe let's simplify like
`contract: { url: 'bla', address: 'bla' }`